x/operator/keeper/consensus_keys.go

```diff
@@ -438,3 +443,113 @@ func (k Keeper) ClearPreviousConsensusKeys(ctx sdk.Context, chainID string) {
 		store.Delete(iterator.Key())
 	}
 }
+
+func (k *Keeper) SetAllPrevConsKeys(ctx sdk.Context, prevConsKeys []types.PrevConsKey) error {
+	store := ctx.KVStore(k.storeKey)
+	for i := range prevConsKeys {
+		prevKey := prevConsKeys[i]
+		keys, err := assetstypes.ParseJoinedStoreKey([]byte(prevKey.Key), 2)
+		if err != nil {
+			return err
+		}
+		chainID := keys[0]
+		opAccAddr, err := sdk.AccAddressFromBech32(keys[1])
+		if err != nil {
+			return err
+		}
+		wrappedKey := types.NewWrappedConsKeyFromHex(prevKey.ConsensusKey)
+		bz := k.cdc.MustMarshal(wrappedKey.ToTmProtoKey())
+
+		store.Set(types.KeyForChainIDAndOperatorToPrevConsKey(chainID, opAccAddr), bz)
+	}
+	return nil
+}
+
+func (k *Keeper) GetAllPrevConsKeys(ctx sdk.Context) ([]types.PrevConsKey, error) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), []byte{types.BytePrefixForOperatorAndChainIDToPrevConsKey})
+	iterator := sdk.KVStorePrefixIterator(store, []byte{})
+	defer iterator.Close()
+
+	ret := make([]types.PrevConsKey, 0)
+	for ; iterator.Valid(); iterator.Next() {
+		var consKey tmprotocrypto.PublicKey
+		k.cdc.MustUnmarshal(iterator.Value(), &consKey)
+		chainID, operatorAddr, err := types.ParsePrevConsKey(iterator.Key())
+		if err != nil {
+			return nil, err
+		}
+		ret = append(ret, types.PrevConsKey{
+			Key:          string(assetstypes.GetJoinedStoreKey(chainID, operatorAddr.String())),
+			ConsensusKey: hexutil.Encode(consKey.GetEd25519()),
```

Comment:
I have created an `operatortypes.WrappedConsKey` structure, which may be useful for this encoding and decoding, as well as conversion of the key into different formats.

Reply:
Thanks, I will update it in the next commit.